CHANGELOG.md

```diff
@@ -1,2 +1,8 @@
+- Re-added a dialog to let users know TLS is being provisioned in App Hosting. (#7595)
```

Comment:
Looks like this changelog got out of sync. Can you ensure that these extra lines dont get included after syncing with master?

Reply:
fixed